http/middleware_test.go

```diff
@@ -13,6 +13,29 @@ import (
 )
 
 func TestWithHeaderAuthorization_InvalidAuthorization(t *testing.T) {
+	kid := "kid-" + t.Name()
+	// Mock the Clerk API server. We expect requests to GET /jwks.
+	clerkAPI := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path == "/jwks" && r.Method == http.MethodGet {
```

Comment:
API versioning is now determined by [a header or query string parameter](https://clerk.com/docs/backend-requests/versioning/overview#choosing-an-api-version).

As a result, adding the version in the path has no effect and might change in the future.

We decided to think of "v1" as part of the API's base URL, since in the future API endpoint paths are likely to omit it.

https://github.com/clerk/clerk-sdk-go/pull/276

Reply:
Resolved offline, the test Client is not configured with the `v1/` path prefix, so this is the request path it accesses.